packages/create-remix/templates/fly-stack/package.json

```
@@ -0,0 +1,34 @@
+{
+  "private": true,
+  "name": "remix-app-template",
+  "description": "",
+  "license": "",
```

Let's add a seed script for prisma to seed the DB with a user and some notes

```suggestion
  "license": "",
  "prisma": {
    "seed": "node --require esbuild-register prisma/seed.ts"
  },
```

(This will require installing `esbuild-register` as a dev dep and creating the `prisma/seed.ts` file.